packages/remix-dev/__tests__/defineRoutes-test.ts

```diff
@@ -95,18 +95,72 @@ describe("defineRoutes", () => {
       route("/other", "routes/other-route.tsx");
     });
 
-    expect(Object.entries(routes)).toHaveLength(3);
```

Comment:
Made this test a bit stronger by asserting that we're getting the right paths, files,  and IDs on defined routes 👍